dcos_test_utils/dcos_cli.py

```diff
@@ -12,16 +12,18 @@
 import stat
 import subprocess
 import tempfile
-from typing import Optional
+from typing import Optional, List
 
 import requests
 
 log = logging.getLogger(__name__)
 
-DCOS_CLI_URL = os.getenv('DCOS_CLI_URL', 'https://downloads.dcos.io/cli/releases/binaries/dcos/linux/x86-64/latest/dcos')  # noqa: E501
+DCOS_CLI_URL = os.getenv('DCOS_CLI_URL', 'https://downloads.dcos.io/cli/releases/binaries/dcos/linux/x86-64/0.8.0/dcos')  # noqa: E501
+CORE_CLI_URL = 'https://downloads.dcos.io/cli/releases/plugins/dcos-core-cli/linux/x86-64/dcos-core-cli-1.13-patch.1.zip'  # noqa: E501
+EE_CLI_URL = 'https://downloads.mesosphere.io/cli/releases/plugins/dcos-enterprise-cli/linux/x86-64/dcos-enterprise-cli-1.13-patch.0.zip'  # noqa: E501
```

Comment:
It'd be nice to have this configurable, to not tie the master branch of dcos-test-utils to DC/OS 1.13. What we had done in the past is pass the CLI URL from dcos-enterprise (https://github.com/mesosphere/dcos-enterprise/pull/4585/files). We could do the same here.

Reply:
done